include/cvc5/cvc5_kind.h

```diff
@@ -539,6 +539,37 @@ enum ENUM(Kind)
    *   - TermManager::mkOp(Kind, const std::vector<uint32_t>&)
    */
   EVALUE(IAND),
+  /**
+   * parametric Integer and.
+   *
+   * \rst
+   * Operator for parametric bit-wise ``AND`` over integers, parameterized by a (positive)
```

Comment:
Maybe state how this is different from IAND. Also it is not necessarily positive, right?